group_vars/all

```diff
@@ -4,6 +4,8 @@ app_package: flaskapp
 app_callable: app
 base_dir: '/srv/{{ app_name }}'
 nginx_server_name: 'localhost {{ app_name }}.local'
+nginx_http_port: 80
```

Comment:
I’ll accept it for port 80 and drop it for port 443. I’ll also clean up the commits and fix git_repo specification.